src/Microsoft.PowerShell.Commands.Management/commands/management/Clipboard.cs

```diff
@@ -157,46 +157,46 @@ public static void SetRtf(string plainText, string rtfText)
         private const uint GMEM_ZEROINIT = 0x0040;
         private const uint GHND = GMEM_MOVEABLE | GMEM_ZEROINIT;
 
-        [DllImport("kernel32.dll")]
-        private static extern IntPtr GlobalAlloc(uint flags, UIntPtr dwBytes);
+        [LibraryImport("kernel32.dll")]
+        private static partial IntPtr GlobalAlloc(uint flags, UIntPtr dwBytes);
 
-        [DllImport("kernel32.dll")]
-        private static extern IntPtr GlobalFree(IntPtr hMem);
+        [LibraryImport("kernel32.dll")]
+        private static partial IntPtr GlobalFree(IntPtr hMem);
 
-        [DllImport("kernel32.dll")]
-        private static extern IntPtr GlobalLock(IntPtr hMem);
+        [LibraryImport("kernel32.dll")]
+        private static partial IntPtr GlobalLock(IntPtr hMem);
 
-        [DllImport("kernel32.dll")]
+        [LibraryImport("kernel32.dll")]
         [return: MarshalAs(UnmanagedType.Bool)]
-        private static extern bool GlobalUnlock(IntPtr hMem);
+        private static partial bool GlobalUnlock(IntPtr hMem);
 
-        [DllImport("kernel32.dll", ExactSpelling = true, EntryPoint = "RtlMoveMemory", SetLastError = true)]
-        private static extern void CopyMemory(IntPtr dest, IntPtr src, uint count);
+        [LibraryImport("kernel32.dll", EntryPoint = "RtlMoveMemory")]
```

Comment:
`LibraryImport` has the `SetLastError` property too, so maybe set that to `true` as well?

Reply:
It is not used in callsite.